en/releases/release-1.5.4.md

```diff
@@ -0,0 +1,19 @@
+---
+title: TiDB Operator 1.5.4 Release Notes
+summary: Learn about new features and bug fixes in TiDB Operator 1.5.4.
+---
+
+# TiDB Operator 1.5.4 Release Notes
+
+Release date: September 13, 2024
+
+TiDB Operator version: 1.5.4
+
+## Improvements
+
+- VolumeReplace feature supports customizing the number of spare replicas used during TiKV or PD disk replacements ([#5666](https://github.com/pingcap/tidb-operator/pull/5666), [@anish-db](https://github.com/anish-db))
+- VolumeReplace feature supports only enabling for specified TiDB clusters ([#5670](https://github.com/pingcap/tidb-operator/pull/5670), [@rajsuvariya](https://github.com/rajsuvariya))
+
```

Comment:
- EBS Snapshot restore supports to configure if warmup failure on individual volumes fails the restore instantly ([#5635](https://github.com/pingcap/tidb-operator/pull/5635), [@michealdeng](https://github.com/[michaelmdeng](https://github.com/michaelmdeng)))
- Gate restore failure on warmup failure only for `check-wal-only` warmup strategy. ([#5636](https://github.com/pingcap/tidb-operator/pull/5636), [@michealdeng](https://github.com/[michaelmdeng](https://github.com/michaelmdeng)))
- support to fetch backup size from backupmeta ([#5430](https://github.com/pingcap/tidb-operator/pull/5430), [@Leavrth](https://github.com/Leavrth))